openhands/runtime/utils/bash.py

```diff
@@ -175,25 +175,29 @@ def __init__(
         work_dir: str,
         username: str | None = None,
         no_change_timeout_seconds: int = 30,
+        max_memory_mb: int = 4 * 1024,  # 4GB
```

Comment:
I'm not sure if it's great to hard-code a limit here. Maybe we could set it to `None` by default and not use a limit when set to none?

Reply:
done!